subprojects/internal-performance-testing/internal-performance-testing.gradle.kts

```diff
@@ -22,9 +22,14 @@ plugins {
 }
 
 val reports by configurations.creating
+val flamegraph by configurations.creating
+configurations.compileOnly.extendsFrom(flamegraph)
 
 repositories {
     javaScript.googleApis()
+    repositories {
+        maven { url = uri("https://jitpack.io") }
```

Comment:
Can we avoid using jitpack here?

Reply:
What's the alternative for getting this integrated without too much ceremony? We could inline the code into Gradle, but then it will only build on Oracle JDK.